Track Buddy/New Group/AccelerometerGraph.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AccelerometerGraph.swift
+//  Track Buddy
+//
+//  Created by Daniel Luo on 6/23/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct AccelerometerGraph: View {
+    @ObservedObject private(set) var motionManager: MotionManager
+    
+    var body: some View {
+        GeometryReader { geometry in
+            let bounds = min(geometry.size.width, geometry.size.height)
+            let pointScaleFactor = -bounds / outerEdgeGValue
```

Comment:
A comment could be helpful to explain this one. Because I (and your future self) may be dumb :p